File: src/main/java/boj/problems/step2/No2753.java
package boj.problems.step2;

import java.io.BufferedReader;
import java.io.BufferedWriter;
import java.io.IOException;
import java.io.InputStreamReader;
import java.io.OutputStreamWriter;

public class No2753 {
    public static void main(String[] args) throws IOException {
        BufferedReader input = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter output = new BufferedWriter(new OutputStreamWriter(System.out));

        output.write(solve(input) + "\n");

        input.close();
        output.flush();
        output.close();
    }

    static int solve(BufferedReader input) throws IOException {
        int year = Integer.parseInt(input.readLine());

        if (year % 4 == 0 && (year % 100 != 0 || year % 400 == 0)) {
            return 1;
        }
        return 0;
    }
}

File: src/main/kotlin/boj/problems/step2/No2753.kt
package boj.problems.step2

import java.io.BufferedReader

object No2753 {
    fun solve(input: BufferedReader): String {
        val year = input.readLine().toInt()
        return if (year % 4 == 0 && (year % 100 != 0 || year % 400 == 0)) {
            "1"
        } else {
            "0"
        }
    }
}
